README.adoc

```diff
@@ -328,11 +328,18 @@ server:
 in_cluster: (true\|false)
 ----
 
-|`SERVICE_FILTER_LABEL_NAME`
-|Label name which all resources of a service are group by. (default is `app`)
+|`APP_FILTER_LABEL_NAME`
```

Comment:
Ah, so we have actually it configurable, cool! I guess a whole check of our codebase would be necessary to check if it's used everywhere it should. (not for this PR)

BTW, wouldn't it be simpler to call it "APP_LABEL_NAME" , without "FILTER" ? (and same for VERSION_LABEL_NAME)

Reply:
+1 , I thought the same thing when I saw these names.

Reply:
It would, but I just copied what was already in place.  I have no objection to changing it.  I'll make the change if there are other things to do for this PR, otherwise in the next PR...

Reply:
I had to make other changes so I did the simplifying rename of the two config settings.